src/components/collection/CollectionForm.tsx

```diff
@@ -311,7 +310,7 @@ export default function CollectionForm({
         {` ${creation ? "Create" : "Update"} collection`}
       </button>
       {" or "}
-      <Link to="/">Cancel</Link>
+      <a href="/#/">Cancel</a>
```

Comment:
Out of curiosity, why this change?

Reply:
Originally it was because we don't need a Link there to do stuff and it was messing with my unit tests. But now I have a better idea that I'm going to implement.